File: 3.4.c
#include<stdio.h>
int main(){

 int l;
 
    printf("the langh of array ???\n ");
    scanf("%d",&l);
int t[l];
    printf (" enter the matrix \n ");
    for (int i = 0;i<l;i++){
        scanf("%d",&t[i]);
    }
     printf ("\n°°° Inverse matrix °°°\n");
for (int i=l-1;i>=0;i--){
    printf ("%d \n ",t[i]);
}return 0;
}

File: 3.4.py
t=[]
l=int(input (" The length of the array?"))
for m in range (l):
    t.append(int(input()))
print ("\n°°° Inverse Array °°°")
for m in range (l-1,-1,-1):
    print(t[m])
    print ('\n')